collections/_article/godot-4-2-arrives-in-style.md

```diff
@@ -0,0 +1,426 @@
+
+- The new `AnimationMixer` node is added as an intermediate class for `AnimationPlayer` and `AnimationTree` ([GH-80813](https://github.com/godotengine/godot/pull/80813)).
+  - This brings both existing nodes to a parity in features and behavior. For the most part this change should not affect existing projects. However, the capture update mode for tracks is no longer available due to various issues in the original implementation. We will be working on providing this or similar functionality in a future version of Godot, but for now you have to work around this limitation using tweens.
+- `GraphEdit` and `GraphNode` nodes are reworked with multiple renames, temporary removal of comment nodes, and introduction of `GraphElement` ([GH-79307](https://github.com/godotengine/godot/pull/79307), [GH-79308](https://github.com/godotengine/godot/pull/79308), [GH-79311](https://github.com/godotengine/godot/pull/79311), [GH-81582](https://github.com/godotengine/godot/pull/81582)).
+  - For the time being, these nodes remain marked as experimental to gather further feedback from users.
+- Several changes in the GLTF importer ([GH-80270](https://github.com/godotengine/godot/pull/80270), [GH-81264](https://github.com/godotengine/godot/pull/81264)) can lead to nodes in imported scenes having slightly different names, compared to Godot 4.1 and before.
+  - Existing imported scenes will remain in the compatibility mode with their node names intact, until you decide to upgrade and reimport them.
+- Changes to the way the engine stores meshes internally have been implemented to improve performance and bandwidth utilization on lower end devices ([GH-81138](https://github.com/godotengine/godot/pull/81138)).
+  - This is done with a grace system in place — existing meshes from Godot 4.1 and prior will be loaded and converted on the fly. There is also the new **Upgrade Mesh Surfaces** tool in the **Project > Tools** menu which will perform a one-time permanent conversion of all meshes in your project, which we recommend you to do.
+- An optimization that required splitting raster barriers into vertex and fragment components changes relevant enumerations ([GH-77420](https://github.com/godotengine/godot/pull/77420)).
+  - It's unlikely that this affects your project, unless you rely on specific numeric values and store them somewhere.
+- An internal rework of the `change_scene_to_*` methods introduces a change in the behavior compared to Godot 4.1 ([GH-78988](https://github.com/godotengine/godot/pull/78988)).
+  - While previously the transition would happen at the end of the current frame, starting with Godot 4.2 the current scene is removed from the tree immediately. This means there is a brief period when no `current_scene` is present in the tree, so you may need to adjust your scripts to account for that. See the updated documentation for more details ([GH-85279](https://github.com/godotengine/godot/pull/85279)).
```

Comment:
```suggestion
  - While previously the transition would happen at the end of the current frame, starting with Godot 4.2 the current scene is removed from the tree immediately. This means there is a brief period when no `current_scene` is present in the tree, so you may need to adjust your scripts to account for that. See the [updated documentation](https://docs.godotengine.org/en/latest/classes/class_scenetree.html) for more details ([GH-85279](https://github.com/godotengine/godot/pull/85279)).
```
I'm not 100% sure this link is ideal, as `latest` will change in the future. If the new docs will be released at the same time as this PR, then pointing to `current` would be better I guess. In any case, I thought it would be nice to link to the docs, seeing as the user is being directed to them.

Reply:
Mind you I've just noticed the link to `latest` docs in the Android section below so maybe that's OK here too.

Reply:
The latest link is fine, we don't have anything better until the version is released.